paasta_tools/cli/cli.py

```diff
@@ -84,6 +84,10 @@ def get_argparser():
     )
 
     subparsers = parser.add_subparsers(help="[-h, --help] for subcommand help")
+    subparsers.required = True
+    # https://stackoverflow.com/questions/18282403/argparse-with-required-subcommands/18283730#18283730
+    subparsers.dest = 'command'
```

Comment:
You can move `dest='command'` as a kwarg to `add_subparsers`, but you won't be able to move `.required = ...` until then.